qcfractal/alembic/versions/a2f76bb7be65_updates_collections_table_with_owner_.py

```diff
@@ -19,7 +18,8 @@
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
     op.add_column('collection', sa.Column('description', sa.String(), nullable=True))
-    op.add_column('collection', sa.Column('owner', sa.String(length=100), nullable=True))
+    op.add_column('collection', sa.Column('group', sa.String(length=100), nullable=True))
+    op.excecute("UPDATE collection SET group=default")
```

Comment:
After the execute add this so there must be a group: `op.alter_column('collection', 'group', nullable=False)`